pf9/modules/express.py

```diff
@@ -178,6 +178,16 @@ def active_config(self):
                     self.ctx.params['du_password'] = config["os_password"]
                     self.ctx.params['du_tenant'] = config["os_tenant"]
                     self.ctx.params['du_region'] = config["os_region"]
+                    # This is needed if we want them to be loaded as python bool
+                    if config.get("dev_key") == 'True':
+                        self.ctx.params['dev_key'] = True
+                    else:
+                        self.ctx.params['dev_key'] = False
```

Comment:
Should we treat configs that don't have this setting as using dev key or prod key? Same question for disabling analytics. Maybe a question for Chris.

Reply:
I would think as dev-key and disable analytics are dev related configs, its better to keep them unset by default. Because, otherwise we need to ask the regular(customer) user to set this flags, which I think would seem unnecessary.